README.md

```diff
@@ -0,0 +1,15 @@
+## git.github.io
+
+- This Repo contains Soc and Outreachy projects of past years.
```

Comment:
I might be Ok with adding a README.md, but it's not true that this repo is just about GSoC and Outreachy. I think the README should contain more or less what there is on: https://git.github.io/

Reply:
@chriscool Kindly Review Now.